pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -21,6 +21,8 @@ in
 
   control-lock = callPackage ./manual-packages/control-lock { };
 
+  copilot-el = callPackage ./manual-packages/copilot-el { };
```

Comment:
This should just be copilot. https://github.com/copilot-emacs/copilot.el/blob/main/copilot.el#L862